lib/userProfileClient.js

```diff
@@ -2,69 +2,67 @@
 
 /**
  * User Profile Service Client
- *
  */
- const fetch = require('fetch-retry-or-die');
- const log = require('@financial-times/n-logger').default;
- const config = require('./config');
- const statusErrors = require('./statusErrors');
- const qs = require('querystring');
+const fetch = require('fetch-retry-or-die');
+const log = require('@financial-times/n-logger').default;
+const config = require('./helpers/config');
+const clientErrors = require('./clientErrors');
+const qs = require('querystring');
+const helpers = require('./helpers/helpers');
 
- const options = Object.assign({}, config.fetchOptions, { headers: Object.assign({}, config.fetchOptions.headers, { "X-API-KEY": config.acqCtxKey })});
+const options = Object.assign({}, config.fetchOptions);
+options.headers["X-API-KEY"] = config.acqCtxKey;
 
-  module.exports = {
- 	 getUUID,
-	 exists
-	//  getProducts,
-	//  getSubsStatus,
- };
+/**
+ * Gets the uuid for a given users email address
+ * @param {String} email - of the user
+ * @return {Promise} response - the user profile with only a uuid value set
+ */
+function getUUID(email) {
+  const operation = 'userProfileClient.getUUID';
+  log.debug({operation, email});
+  const queryString = qs.stringify({email});
+  const url = `${config.userProfileURL}?${queryString}`;
+  return fetch(url, options)
+    .then(res => helpers.parseJsonRes(res, `${operation} ${queryString}`))
+    .then(body => {
+      if (Array.isArray(body.items)) {
```

Comment:
Since we refer to `body.items` a few times here, could we cache it into a variable?